BinInfoUtils.h

```diff
@@ -0,0 +1,55 @@
+#ifndef _bininfoutils_
+#define _bininfoutils_
+
+// could #include "TMath.h", but then this is problematic for non root running.  This compiles and runs without tmath.h
+
+#include <cmath>
+#include <vector>
+#include "Config.h"
+#include <iostream>
+
+typedef std::pair<unsigned int,unsigned int> BinInfo;
+typedef std::vector<std::vector<BinInfo> > BinInfoLayerMap;
+typedef std::vector<std::vector<std::vector<BinInfo> > > BinInfoMap;
+
+inline float downPhi(float phi){
+  while (phi >= Config::PI) {phi-=Config::TwoPI;}
+  return phi;
+}
+	
+inline float upPhi(float phi){
+  while (phi <= -Config::PI) {phi+=Config::TwoPI;}
+  return phi;
+}
+
+inline float normalizedPhi(float phi) {
+  //  return std::fmod(phi, (float) Config::PI); // return phi +pi out of phase for |phi| beyond boundary! 
+  if (fabs(phi)>=Config::PI) {phi = (phi>0 ? downPhi(phi) : upPhi(phi));}
```

Comment:
I changed only the fabs() in BinInfoUtils.h to std::abs().  As we discussed in the meeting, we will need to make a commit fully changing all fabs() and abs() to std::abs(), as well as sqrt, sqrtf, etc.